### ISSUE TITLE
 D3D11 Compatibility Issue

### ISSUE THREAD

**Issue by LEM1OUTGAMES** [doitsujin/dxvk#4299]
Hello Philip,

I’m a game developer who has worked on 
![image](https://github.com/user-attachments/assets/b4c92ba8-d883-400b-bc6b-1bb8d7181e0d)
couple games, but I’ve encountered a problem with D3D11. I’m not sure why this is happening, but I have an old DXVK image. I’ve looked into various solutions, such as removing DXVK and trying different DLLs from [dll-files.com](https://www.dll-files.com/), but nothing seems to work correctly.

I hope you might know a way to resolve this issue. Thank you for taking the time to read my message, 

Best regards,
Jacob Jufeili

**Comment by Blisto91** [comment]
I am not Philip but wanted to ask for some clarification 

> I’ve looked into various solutions, such as removing DXVK and trying different DLLs from dll-files.com, but nothing seems to work correctly.

Are you saying this is a issue with the game even when not using dxvk?

**Comment by doitsujin** [comment]
32768 is not a supported texture size, the maximum size commonly supported by current hardware is 16384. This isn't expected to work anywhere.

**Comment by LEM1OUTGAMES** [comment]
So what could cause that issue for some reason if do change the value on the Notepad and save it as 16384 or less is going to read it has 32768

**Comment by LEM1OUTGAMES** [comment]
> I am not Philip but wanted to ask for some clarification
> 
> > I’ve looked into various solutions, such as removing DXVK and trying different DLLs from dll-files.com, but nothing seems to work correctly.
> 
> Are you saying this is a issue with the game even when not using dxvk?

Without DXVK, the game won't run. You can download the game as a demo on Steam. I have it there; it's called the Nightmare of Arrival chapter 1 demo, but with Nvidia works fine.

**Comment by LEM1OUTGAMES** [comment]
I think I may have found the issue. I was having a problem with NVIDIA. When I removed DXVK, the game didn’t work. So, I started to go through each file until I found the Fonts folder. I deleted most of the contents there, and it worked fine without DXVK. However, when I put the files back, the game crashed. I believe I’ve identified my issue, and this might be why it breaks on AMD cards. I plan to fix this when I set up my second PC and install the AMD card in it. If that works, then it’s likely the fonts that are causing the problem.
